internal/api/token_test.go

```diff
@@ -60,6 +60,40 @@ func (ts *TokenTestSuite) SetupTest() {
 	require.NoError(ts.T(), err, "Error creating refresh token")
 }
 
+func (ts *TokenTestSuite) TestSessionTimebox() {
+	timebox := 10 * time.Second
+
+	ts.API.config.Sessions.Timebox = timebox
+	ts.API.overrideTime = func() time.Time {
+		return time.Now().Add(timebox).Add(time.Second)
+	}
+
+	defer func() {
+		ts.API.overrideTime = nil
+	}()
+
+	var buffer bytes.Buffer
+	require.NoError(ts.T(), json.NewEncoder(&buffer).Encode(map[string]interface{}{
+		"refresh_token": ts.RefreshToken.Token,
+	}))
+
+	req := httptest.NewRequest(http.MethodPost, "http://localhost/token?grant_type=refresh_token", &buffer)
+	req.Header.Set("Content-Type", "application/json")
+
+	w := httptest.NewRecorder()
+	ts.API.handler.ServeHTTP(w, req)
+	assert.Equal(ts.T(), http.StatusBadRequest, w.Code)
+
+	var firstResult struct {
+		Error            string `json:"error"`
+		ErrorDescription string `json:"error_description"`
+	}
+
+	assert.NoError(ts.T(), json.NewDecoder(w.Result().Body).Decode(&firstResult))
+	assert.Equal(ts.T(), firstResult.Error, "invalid_grant")
```

Comment:
/nit Think the [expected argument comes first](https://pkg.go.dev/github.com/stretchr/testify/assert#Equal)
```suggestion
	assert.Equal(ts.T(), "invalid_grant", firstResult.Error)
```